docs/source/installation.rst

```diff
@@ -54,7 +54,7 @@ It is recommended to install the same version as the one used by torch.
 
 .. code-block:: shell
 
-   conda install -c conda-forge cuda-nvcc cuda-libraries-dev cuda-version "gxx<12" pytorch=*=*cuda*
+   conda install -c conda-forge cuda-nvcc cuda-libraries-dev cuda-version "gxx<12" "pytorch=*=*cuda*"
```

Comment:
also fixed this because it bothered me :)

Reply:
The " surrounds gxx because otherwise bash will interpret it as "my_command<12" (a.i. pipe the file "12" as input). I have never found it a problem to not add the quotes when using wildcards. But I can see it being a problem.

Reply:
```sh
(torchmd-net2)  yyy@xxx  ~/Work/torchmd-net   versioning ±  mamba install -c conda-forge cuda-nvcc cuda-libraries-dev cuda-version "gxx<12" pytorch=*=*cuda*
zsh: no matches found: pytorch=*=*cuda*
```
might just be my `zsh` but it doesn't work for me